src/main/java/jalau/at18/kingoftokyo/DiceFaceIndentifier.java

```diff
@@ -2,13 +2,13 @@
 
 import java.util.HashMap;
 
-public class DiceIndentifier {
+public class DiceFaceIndentifier {
     private HashMap<DiceFace, Integer> diceCount;
 
-    public DiceIndentifier() {
+    public DiceFaceIndentifier() {
         this.diceCount = new HashMap<DiceFace, Integer>();
```

Comment:
You could just leave this line like 
`this.diceCount = new HashMap<>();` 
because you already declare it up top with HashMap<DiceFace, Integer>

Reply:
ok